events/management/commands/check_deprecated_yso_keywords.py

```diff
@@ -0,0 +1,88 @@
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+
+from events.importer import yso
+from events.models import Keyword
+
+
+class DryRun(Exception):
+    pass
+
+
+class Command(BaseCommand):
+    help = "Check replacements of deprecated keywords."
```

Comment:
"Check" would imply a safe operation; however, this is only a check if you run it dry. Maybe rename to `replace_deprecated_yso_keywords`? Or something similar that makes it clear that this *will* alter data.

Reply:
I've updated the help text to read "Check or apply". The default operation is to check only.